R/data.R

```diff
@@ -21,6 +24,10 @@ NULL
 #' 
 #' This is a subset of 200 observations from the 2010 cycle of the CCHS PUMF dataset. 
 #' 
+#' Long name: CCHS-82M0013-E-2010-AnnualComponent
```

Comment:
Looks all good. I suggest that you add the rest of the years/data sets (at the risk of lots of cut and pasting).